tests/release_notes/test_release_notes_builder.py

```diff
@@ -513,121 +516,142 @@ def test_build_no_data_no_empty_chapters():
     #   - covered in 'test_build_merged_pr_service_chapter_without_issue_and_user_labels'
 
 
-def test_build_closed_issue_with_one_custom_label(custom_chapters_not_print_empty_chapters,
-                                                  record_with_issue_closed_two_pulls):
+def test_build_closed_issue_with_one_custom_label(
+        custom_chapters_not_print_empty_chapters, record_with_issue_closed_two_pulls, mocker):
     expected_release_notes = release_notes_data_custom_chapters_one_label
     rec = record_with_issue_closed_two_pulls
+    mocker.patch('release_notes_generator.builder.ActionInputs.get_print_empty_chapters', return_value=False)
+
     builder = ReleaseNotesBuilder(records={rec.number: rec},
                                   changelog_url=default_changelog_url,
                                   formatter=default_formatter,
                                   custom_chapters=custom_chapters_not_print_empty_chapters,
-                                  print_empty_chapters=False)
+                                  )
 
     actual_release_notes = builder.build()
 
     assert expected_release_notes == actual_release_notes
 
 
-def test_build_closed_issue_with_more_custom_labels_duplicity_reduction_on(custom_chapters_not_print_empty_chapters,
-                                                                           record_with_issue_closed_two_pulls):
+def test_build_closed_issue_with_more_custom_labels_duplicity_reduction_on(
+        custom_chapters_not_print_empty_chapters, record_with_issue_closed_two_pulls, mocker):
     expected_release_notes = release_notes_data_custom_chapters_more_labels_duplicity_reduction_on
     rec = record_with_issue_closed_two_pulls
     rec.issue.labels.append(MockLabel("enhancement"))
     rec.issue.title = 'I1+bug-enhancement'
+    mocker.patch('release_notes_generator.builder.ActionInputs.get_print_empty_chapters', return_value=False)
+
     builder = ReleaseNotesBuilder(records={rec.number: rec},
                                   changelog_url=default_changelog_url,
                                   formatter=default_formatter,
                                   custom_chapters=custom_chapters_not_print_empty_chapters,
-                                  print_empty_chapters=False)
+                                  )
 
     actual_release_notes = builder.build()
 
     assert expected_release_notes == actual_release_notes
 
 
 def test_build_closed_issue_service_chapter_without_pull_request_and_user_defined_label(
-        custom_chapters_not_print_empty_chapters, record_with_issue_closed_no_pull):
+        custom_chapters_not_print_empty_chapters, record_with_issue_closed_no_pull, mocker):
```

Comment:
Move :) into new line like it is done in all other methods.

Reply:
Fixed on commit TODO.